src/assets/database.sql

```diff
@@ -366,6 +366,44 @@ INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `comm
 ('Zerocopy', '(client only) Use a "zero copy" method of sending data', 'iperf3 --zerocopy', 19, 'checkbox', '--zerocopy', NULL),
 ('Dont fragment', '(client only) Set the IPv4 Dont Fragment (DF) bit on outgoing packets', 'iperf3', 19, 'checkbox', '--dont-fragment', NULL);
 
+--
+-- Populate data for `commands`: dig
+--
+INSERT INTO `commands` (`name`, `description`, `examples`, `tool`, `type`, `command`, `example`, `sudo`, `category`) VALUES 
+('DNS Server', 'Defines the DNS server which will be used in the command', 'dig @8.8.8.8 example.hackinsdn.ubfa,br', 27, 'input', NULL, NULL, NULL, 'BASIC OPTIONS'),
```

Comment:
I'm not sure this will work. To set the dns server dig to use it must be preceeded by `@`.

Reply:
Hey, @mayara-santos01 , please take a look at this point. It would be interesting to set DNS server to use while executing `dig` through Secflood, but this option is missed from the `dig` commands. What about to add it?

Reply:
yes, this was removed. In order to specify the server, one can type in the main "target" input something like "@172.16.50.1 xpto.com". Please let me know what you think.